Library/Homebrew/language/python.rb

```diff
@@ -215,6 +215,14 @@ def create
             f.unlink
             f.make_symlink new_target
           end
+
+          Pathname.glob(@venv_root/"lib/python*/orig-prefix.txt").each do |prefix_file|
+            prefix_path = prefix_file.read
+            next unless prefix_path.start_with? HOMEBREW_CELLAR
+            python = prefix_path.include?("python3") ? "python3" : "python"
+            prefix_path.sub! %r{#{HOMEBREW_CELLAR}/#{python}/[^/]+}, Formula[python].opt_prefix
```

Comment:
is this going to break if the requirement is fulfilled by non-brew python/python3?